usr/share/rear/prep/BAREOS/default/500_check_BAREOS_bconsole_results.sh

```diff
@@ -3,39 +3,41 @@
 # are all  the files/directories present?
 
 if [ "$BEXTRACT_DEVICE" -o "$BEXTRACT_VOLUME" ]; then
-
-   ### Bacule restore using bextract, no need for a director
+   ### restore using bextract, no need for a director
    return
 fi
 
-# Check bconsole version and use appropriate CLI switch:
-# Bareos 22 introduced a breaking change how its CLI tools (such as bconsole) parse the arguments.
-# Before it was "bconsole -xc" but since Bareos 22 it must be "bconsole --xc"
-# (otherwise it terminates with exit code 113 so "rear mkrescue" will fail).
-# Before "bconsole ...xc" is called, it now checks the bconsole version
-# and uses the new argument format '--xc' when version >= 22.
-# The backward compatible fallback is "bconsole -xc".
-# See https://github.com/rear/rear/issues/2900
-BCONSOLE_VERSION=$(bconsole --version | awk -F '.' '{print $1}')
-if [ "$BCONSOLE_VERSION" -ge 22 ]; then
-	BCONSOLE_XC="--xc"
-else
-	BCONSOLE_XC="-xc"
-fi
+if [ "$PING" ]; then
```

Comment:
Caution because is_true and is_false
do not implement strict boolean semantics,
see the is_true and is_false description
in usr/share/rear/lib/global-functions.sh
currently starting at
https://github.com/rear/rear/blob/master/usr/share/rear/lib/global-functions.sh#L124

Reply:
I removed the whole PING section, as it causes some problems and don't bring much benefit.